spec/features/trainer_spec.rb

```diff
@@ -1,313 +1,179 @@
-require 'rails_helper'
-require 'support/helpers/login_helper.rb'
+require "rails_helper"
+require "support/helpers/login_helper.rb"
 include LoginHelper
 
-describe 'review cards without blocks' do
-  describe 'training without cards' do
-    before do
-      create(:user)
-      visit trainer_path
-      login('test@test.com', '12345', 'Войти')
-    end
-
-    it 'no cards' do
-      expect(page).to have_content 'Ожидайте наступления даты пересмотра.'
+describe "review cards" do
+  # let!(:user) { create(:user, locale: 'ru') }
+  let!(:user) { User.create(email: 'test@test.com', password: '12345', password_confirmation: '12345', locale: 'ru') }
```

Comment:
Line is too long. [118/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.